tests/v2/validation/provisioning/registries/registry_test.go

```diff
@@ -207,55 +232,71 @@ func (rt *RegistryTestSuite) TestRegistriesRKE() {
 	subSession := session.NewSession()
 	defer subSession.Cleanup()
 
-	subClient, err := rt.client.WithSession(subSession)
+	subClient, err := rt.standardUserClient.WithSession(subSession)
 	require.NoError(rt.T(), err)
 
+	if rt.provisioningConfig.RKE1KubernetesVersions == nil {
+		rke1Versions, err := kubernetesversions.ListK3SAllVersions(rt.client)
+		require.NoError(rt.T(), err)
+
+		rt.provisioningConfig.RKE1KubernetesVersions = rke1Versions
+	}
+
 	tests := []struct {
 		registry []management.PrivateRegistry
 		name     string
 	}{
-		{rt.privateRegistriesNoAuth, "RKE1 Registry No Auth"},
-		{rt.privateRegistriesAuth, "RKE1 Registry Auth"},
-		{rt.privateEcr, "RKE1 Registry ECR"},
+		{rt.privateRegistriesNoAuth, "RKE1 Registry No Auth "},
+		{rt.privateRegistriesAuth, "RKE1 Registry Auth "},
+		{rt.privateEcr, "RKE1 Registry ECR "},
```

Comment:
What's the reason for adding the space at the end?

Reply:
I'm pretty sure this is for Qase and the naming.

Reply:
correct